### PR TITLE
Update rackt reference to reactjs

### PR DESCRIPTION
**Pull request by cwheikki**
Update rackt references to reactjs.

Addresses comments at https://github.com/reactjs/react-redux/pull/299

### CODE REVIEW

**Comment by cwheikki** [comment]
Also - I've been using react-redux for a few months now. I must say, I am very impressed! There are many talented people in this community. Thanks for all of the great work everyone has put into it. I am looking forward to deploying our first production application in coming weeks!

Cheers,
Chris

File: docs/api.md [hunk 1]
```diff
@@ -53,18 +53,18 @@ ReactDOM.render(
 
 Connects a React component to a Redux store.
 
-It does not modify the component class passed to it.
```
Comment (gaearon) [inline docs/api.md]: Whitespace is significant in markdown. By removing these two spaces, you are changing how lines are wrapped.

File: docs/api.md [hunk 2]
```diff
@@ -108,8 +108,8 @@ export default connect()(TodoApp)
 
 ##### Inject `dispatch` and every field in the global state
 
->Don’t do this! It kills any performance optimizations because `TodoApp` will rerender after every action.
```
Comment (gaearon) [inline docs/api.md]: Same here (and several times above). Please don’t remove the whitespace :wink:

**Comment by gaearon** [comment]
:+1: Thank you!

**Pull request merged**